examples/tiny_skia_render/src/main.rs

```diff
@@ -96,7 +96,11 @@ fn main() {
 
     // Perform layout (including bidi resolution and shaping) with start alignment
     layout.break_all_lines(max_advance);
-    layout.align(max_advance, Alignment::Start);
+    layout.align(
+        max_advance,
+        Alignment::Start,
+        false, /* align_when_overflowing */
```

Comment:
If `align` gains more options in the future, a boolean may be a bit confusing, especially if people don't document as you've done here. Maybe an `AlignmentOptions` struct could be introduced?

Reply:
Agree, but don't have time to work on this now. In the interest of getting this landed for Parley 0.3 I'm going to merge this as-is. And have created an issue for the options struct (https://github.com/linebender/parley/issues/247)